crd-convert/src/main/java/io/strimzi/kafka/crd/convert/cli/ExitCommand.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.crd.convert.cli;
+
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "exit", aliases = {"x"}, description = "Exit shell")
```

Comment:
Oh, we've got a CLI which operates as a shell? Why?

Reply:
It can do both -- plain single command line or full shell.
I added shell b/c it's really cool. :-)